sycl/include/CL/sycl/version.hpp

```diff
@@ -0,0 +1,12 @@
+//==------ version.hpp --- SYCL compiler version macro ---------*- C++ -*---==//
```

Comment:
This is auto-generated file which generates in `build` directory. It shouldn't be commited to the repo.

Reply:
oh, thanks. deleted.